src/main.js

```diff
@@ -345,6 +344,28 @@ function singleClickEvent(e) {
     }
   }
 
+  // Message Passing to the Container Code. Package include the id & label
+  if (window.parent) {
```

Comment:
Please put the message passing interface in its own function that is called by SelectNode